db/queries/exec_volumes.sql

```diff
@@ -0,0 +1,10 @@
+-- name: ExecVolumeGet :many
+select exec_id, volume_id, mount_path from unweave.exec_volume as ev where ev.exec_id = $1;
+
+-- name: ExecVolumeCreate :exec
+insert into unweave.exec_volume (exec_id, volume_id, mount_path)
+values ($1, $2, $3) on conflict do nothing;
```

Comment:
Not sure about the `conflict do nothing` part. We should probably throw an error so this doesn't do something unexpected. 

Also I realized the unique constraint is execID and volumeID. This means you can't mount the same volume twice at two different mount_paths on the same exec. That seems an unnecessary constraint. 

So we probably want to update the constraint to be on all three columns and fail on conflict.

Reply:
I looked into this and the uniqueness constraint is on execID, volumeID, and mount path